cmd/clusterctl/api/v1alpha3/labels.go

```diff
@@ -19,31 +19,32 @@ package v1alpha3
 import "fmt"
 
 const (
-	// ClusterctlLabelName defines the label that is applied to all the components managed by clusterctl.
+	// ClusterctlLabelName is applied to all components managed by clusterctl.
 	ClusterctlLabelName = "clusterctl.cluster.x-k8s.io"
 
-	// ClusterctlCoreLabelName defines the label that is applied to all the core objects managed by clusterctl.
+	// ClusterctlCoreLabelName is applied to all the core objects managed by clusterctl.
 	ClusterctlCoreLabelName = "clusterctl.cluster.x-k8s.io/core"
 
-	// ClusterctlResourceLifecyleLabelName defines the label that documents the lifecyle for a specific resource.
-	// e.g. resources shared between instances of the same provider. e.g. CRDs, ValidatingWebhookConfiguration, MutatingWebhookConfiguration etc.
-	// are marked as shared
+	// ClusterctlResourceLifecyleLabelName describes the lifecyle for a specific resource.
+	//
+	// Example: resources shared between instances of the same provider:  CRDs,
+	// ValidatingWebhookConfiguration, MutatingWebhookConfiguration, and so on.
```

Comment:
The previous comment here attempted to indicate that the value is set based on the "type" of lifecycle the resource has. It might be good to call that out a bit more explicitly with the new edit.

It also looks like below we only define a "shared" lifecycle, should we also have another valid version for lifecycle that indicates that it is per-tenant?

Reply:
> The previous comment here attempted to indicate that the value is set based on the "type" of lifecycle the resource has. It might be good to call that out a bit more explicitly with the new edit.

What do you mean here? Reading the version before/after I don't notice less information, an example would be helpful

> It also looks like below we only define a "shared" lifecycle, should we also have another valid version for lifecycle that indicates that it is per-tenant?

probably out of scope for this PR given that I'm only trying to do some cleanups, but that's probably fine as a follow up